server/routerlicious/packages/services-client/src/historian.ts

```diff
@@ -129,8 +129,12 @@ export class Historian implements IHistorian {
             `/git/trees/${encodeURIComponent(sha)}`,
             this.getQueryString({ recursive: recursive ? 1 : 0 }));
     }
-    public async createSummary(summary: IWholeSummaryPayload): Promise<IWriteSummaryResponse> {
-        return this.restWrapper.post<IWriteSummaryResponse>(`/git/summaries`, summary, this.getQueryString());
+    public async createSummary(summary: IWholeSummaryPayload, initial?: boolean): Promise<IWriteSummaryResponse> {
+        return this.restWrapper.post<IWriteSummaryResponse>(
+            `/git/summaries`,
+            summary,
+            this.getQueryString(initial === undefined ? undefined : { initial }),
```

Comment:
Would this not work? 
```ts
initial ? { initial} : undefined
```
Or since in Historian we check the opposite (`initial !== undefined` as opposed to `initial === undefined`), do you think it is worth having the same check on both places?